kubernetes/rucio/README.md

```diff
@@ -0,0 +1,146 @@
+Adapted from rucio instructions here: https://github.com/rucio/helm-charts/tree/master/rucio-server
+
+
+# First time setup
+
```

Comment:
Two things:
1. please specify the machine users should use, ssh lxplus-cloud

2. I would suggest to get rid of your personal directories in instructions and define here something like:
```
wdir=$PWD # change appropriately to whatever working dir you're comfortable to
```
Then replace `/afs/cern.ch/user/e/ewv` with `$wdir`. This will allow to make instructions to be used as copy-and-paste procedure.